metaci/build/models.py

```diff
@@ -410,6 +418,27 @@ def run(self):
             )
             if org_config.created:
                 self.delete_org(org_config)
+
+            if (
+                self.plan.change_traffic_control
+            ):  # Calling for any actions taken against packaging org
+                try:
+                    send_stop_webhook(
+                        self.release,
+                        self.plan.role,
+                        self.org.configuration_item,
+                    )
+                except Exception as err:
+                    message = f"Error while sending implementation stop step webhook: {err}"
+                    self.logger.error(message)
+                    set_build_info(
+                        build, status="error", exception=message, time_end=timezone.now()
+                    )
+                    self.logger = init_logger(self)
+                    self.logger.error(str(e))
+                    self.delete_build_dir()
+                    self.flush_log()
+                    return
```

Comment:
Let's move this entire block for processing the webhook down between lines 444 and 445. That way:
a. We don't need to call delete_build_dir and flush_log from two different places
b. We'll be sure to log both the error that stopped the build and possibly an error while calling the webhook

Also let's remove set_build_info from this block. An error while sending the stop webhook should not overwrite the main error that stopped the build.

Don't we need to add a parameter to send_stop_webhook so that we can set the status on the implementation step differently based on whether the build succeeded or failed? We don't want it to end up as "Implemented - per plan" if it failed.